src/MultiSelect/index.js

```diff
@@ -176,7 +183,12 @@ const MultiSelect = ({
 
   return (
     <div className="nds-multiselect" data-testid={testId}>
-      <input type="hidden" name={name} id={name} value={fieldValue} />
+      <input
+        type="hidden"
+        name={name}
+        id={name}
+        value={fieldValue || selectedItems.map(itemToString).join(",")}
```

Comment:
`fieldValue` is now an optional prop - it's only responsible for setting the hidden input value, and now it has the default behavior of holding a comma separated list of values